client/src/components/AccountLayout.js

```diff
@@ -1,38 +1,56 @@
 import React from 'react'
 import { Box, Heading, Main } from 'grommet'
 import { useRouter } from 'next/router'
+import { useNotifications } from 'hooks/useNotifications'
+import { NOTIFICATION_TYPES } from 'helpers/notificationTypes'
 import { useIsClient } from '../hooks/useIsClient'
 import { SideNav } from './SideNav'
 import Header from './Header'
 
+const getNotificationsForType = (notifications, type) => {
+  if (!notifications) {
+    return {}
+  }
+  const notifsForType = notifications.filter((notification) => {
+    return NOTIFICATION_TYPES[type].includes(notification.notification_type)
+  })
+
+  return notifsForType
+}
+
 export const AccountLayout = ({ children }) => {
   const isClient = useIsClient()
+  const { getUnreadNotifications } = useNotifications()
   const router = useRouter()
+
+  let unreadNotifs = []
+  unreadNotifs = getUnreadNotifications()
   const links = [
     {
       text: 'Basic Information',
       href: '/account/basic-information',
-      notifications: 0
+      notifications: getNotificationsForType(unreadNotifs, 'basicInfo').length
```

Comment:
we should remove this for everything but the `notifications` sections

Reply:
Gotcha, removed